bgpd/bgp_nexthop.c

```diff
@@ -753,6 +760,22 @@ static void bgp_show_nexthop_paths(struct vty *vty, struct bgp *bgp,
 		safi = table->safi;
 		bgp_path = table->bgp;
 
+		if (json) {
+			json_path = json_object_new_object();
+			json_object_string_add(json_path, "afi", afi2str(afi));
+			json_object_string_add(json_path, "safi",
+					       safi2str(safi));
+			if (dest->pdest)
```

Comment:
Could we add "dest" to JSON output?

Reply:
Do you mean renaming "rd" to "dest"?

Reply:
Along with RD, could we add "dest" to JSON output? vtysh output for "show bgp nexthop x.x.x.x " prints the prefix/dest along with RD:

show bgp nexthop 6.0.0.28 :
<snip>
 Paths:
    3/5 **[3]:[6.0.0.28]/352** RD 6.0.0.28:7 VRF default flags 0x410
    3/5 **[2]:[00:02:00:00:00:0b]:[fe80::202:ff:fe00:b]/352** RD 6.0.0.28:7 VRF default flags 0x410
<snip>

Reply:
Fixed